### PR TITLE
feat(eis): display switching

### LINKED ISSUE #1921: [REQUEST] ECAM - Add switching panel / DMC functionality
**Is your feature request related to a problem? Please describe.**

Improve realism

**Describe the feature you'd like to see implemented**

* Switching the upper ECAM off (using the brightness switch) should show the E/WD (engine/warning) page on the lower ECAM (instead of the system/status page)
* Turning the ECAM/ND XFR (the rightmost) switch to CAPT or F/O should move the SD (system/status) page to the left or right ND panel, replacing the navigation display. If the upper ECAM is switched off and the lower ECAM shows the E/WD page, the E/WD page is moved instead to the appropriate ND display
* Turning off a PFD (using the PFD brightness switch) should display the PFD on the ND display (on the respective side). 
* Pushing the PFD/ND XFR button (between the PFD / ND brightness knobs) should switch the respective PFD / ND displays

**References**

image
image
image
image

**Additional context**

Discord username (if different from GitHub):
n/a

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #1921 

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Nufflee** [comment]
Render version of the DU switching doc for anyone interested: https://github.com/tracernz/a32nx/blob/mrc-cursed-configs/docs/display-unit-switching.md

File: flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
         <Component ID="#NODE_ID#" Node="#NODE_ID#">
             <UseTemplate Name="ASOBO_GT_Push_Button_Held">
-                <LEFT_SINGLE_CODE>0</LEFT_SINGLE_CODE><!-- TODO -->
+                <LEFT_SINGLE_CODE>(L:A32NX_EFIS_PFD_ND_XFR_#ID#, bool) ! (&gt;L:A32NX_EFIS_PFD_ND_XFR_#ID#, bool)</LEFT_SINGLE_CODE><!-- TODO -->
                 <TOOLTIPID>Inoperant</TOOLTIPID>
```
Comment (bouveng) [inline flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml:57]: Remove or add relevant tooltip.

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #6929
Tier of Testing  : 1
Date             : 06/04/2022

Testing Process: 
Tried switching of various display combinations, everything works.

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image
image

**Comment by aguther** [comment]
I've noticed shadow in the cockpit to change when display switching is off/on. What could be the reason for this?

**Comment by tracernz** [comment]
> I've noticed shadow in the cockpit to change when display switching is off/on. What could be the reason for this?

The *HUGE* display just behind the cockpit...
image

**Comment by aguther** [comment]
I close this for now. Can be reopened any time, but my guess would be that at some point in time the approach how this implemented will be different.

**Pull request closed without merging**